src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -171,22 +176,49 @@ public void assertType(@NotNull Material material)
 	@Override
 	public byte getLightLevel()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return lightLevel;
```

Comment:
Looked into the source code of paper, this value is the max value of `lightFromSky` and `lightFromBlocks`. Not it's own field